### PR TITLE
[virustotal] Flag for option to keep higher score of VirusTotal or existing score

### PR DESCRIPTION
**Pull request by sc0ttes**
### Proposed changes

* Environment variable in VirusTotal connector to allow for only replacing observable score with VT score if VT score is higher.
* Add note with VT score to observable if VT score is lower than existing score

### Related issues

* #913 

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality using different use cases
- [x] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

**Comment by axelfahy** [comment]
I don't understand what it has to do with the `confidence_level`, in the PR, you are modifying the score of the observable, not the confidence level.

File: internal-enrichment/virustotal/src/virustotal/builder.py [hunk 1]
```diff
@@ -72,13 +71,20 @@ def _compute_score(stats: dict) -> int:
...
+        vt_score = round(
             (
                 stats["malicious"]
                 / (stats["harmless"] + stats["undetected"] + stats["malicious"])
             )
             * 100
         )
+        if self.replace_with_lower_score:
+            return vt_score
+        self.create_note(
```
Comment (axelfahy) [inline internal-enrichment/virustotal/src/virustotal/builder.py]: I guess the note is to keep track of the VirusTotal score if we don't use it to update the score of the observable. However, if `replace_with_lower_score=false` and `vt_score > current score`, we will update with the VirusTotal score and have the note with the same score. Is it the expected behaviour?

**Comment by sc0ttes** [comment]
@axelfahy , agree -- not sure why the original ticket's title was talking about confidence level but just stuck with it's title. Fixed now.

File: internal-enrichment/virustotal/src/virustotal/builder.py (hunk 1 above)
Comment (sc0ttes) [reply internal-enrichment/virustotal/src/virustotal/builder.py]: Good point -- fixing.

**Pull request merged**